opcua/server/address_space.py

```diff
@@ -463,6 +463,8 @@ def _call(self, method):
                 res.StatusCode = ua.StatusCode(ua.StatusCodes.BadNothingToDo)
             else:
                 try:
+                    if method.InputArguments is None:
+                        method.InputArguments = [] 
```

Comment:
these lines will never fail. No need to have them under try/except

Reply:
These lines will never fail, but the actual method call could. I have put this check in the same scope as the method call for readability. Would you prefer it outside of the try statement?

Reply:
OK I just looked at that code. In fact we should only have try/except  around the node.call line. Can you move everything else out?